packages/webapp/src/routes/land/clients-email-address.js

```diff
@@ -1,13 +1,36 @@
 import constants from '../../utils/constants.js'
+import { validateEmail } from '../../utils/helpers.js'
 
 const handlers = {
   get: async (request, h) => {
-    return h.view(constants.views.CLIENTS_EMAIL_ADDRESS)
+    const email = request.yar.get(constants.redisKeys.CLIENTS_EMAIL_ADDRESS)
+    return h.view(constants.views.CLIENTS_EMAIL_ADDRESS, {
+      email
+    })
+  },
+  post: async (request, h) => {
+    const email = request.payload.email
```

Comment:
I prefer destructing but its up to you :) 
```suggestion
    const { email } = request.payload
```

Reply:
Think I have the habit of destructuring if there are multiple values being pulled, but not if there is only 1, happy to update